src/badguy/walking_badguy.hpp

```diff
@@ -21,9 +21,21 @@
 
 class Timer;
 
+static const int s_normal_max_drop_height = 600;
+
```

Comment:
Put this under `WalkingBadguy` as `protected`.

Reply:
Ok but why?

Reply:
Scoping.